packages/maskbook/src/web3/hooks/useERC20TokenTransferCallback.ts

```diff
@@ -49,9 +52,14 @@ export function useERC20TokenTransferCallback(address: string, amount?: string,
             to: erc20Contract.options.address,
         })
 
+        const gas = new BigNumber(estimatedGas).div(1e9).multipliedBy(new BigNumber(10).pow(token?.decimals || 18))
```

Comment:
Why you need to change this hook? It's all about transferring Ether.

Reply:
reverted